tests/raftstore/test_multi.rs

```diff
@@ -609,3 +609,35 @@ fn test_server_remove_leader_with_uncommitted_log() {
     let mut cluster = new_server_cluster(0, 2);
     test_remove_leader_with_uncommitted_log(&mut cluster);
 }
+
+fn test_follower_read<T: Simulator>(cluster: &mut Cluster<T>) {
+    cluster.cfg.raft_store.raft_election_timeout_ticks = 50;
+    // disable compact log to make test more stable.
+    cluster.cfg.raft_store.raft_log_gc_threshold = 1000;
+    // We use three peers([1, 2, 3]) for this test.
+    cluster.run();
+
+    sleep_ms(500);
+
+    let (k0, v0) = (b"k0", b"v0");
```

Comment:
s/k0, v0/k, v/
Only use suffix number when there could be multiple versions of key-value.